autogen/oai/client.py

```diff
@@ -83,7 +83,7 @@ class Message(Protocol):
         choices: List[Choice]
         model: str
 
-    def create(self, **params: Any) -> ModelClientResponseProtocol:
+    def create(self, params: Dict[str, Any]) -> ModelClientResponseProtocol:
```

Comment:
@olgavrou perhaps the fix is to fix `OpenAIClient`'s `create` signature?

Reply:
The fix provided is sufficient, in fact in all the implementations (test and notebook) we haven't used `**params` but `params`, so this protocol update is essentially a bug fix

Reply:
Okay.